Makefile

```diff
@@ -77,7 +77,7 @@ test-in-docker:
 	# Privileged needed for docker-in-docker so integ tests pass
 	docker run -v "$(shell pwd):/go/src/github.com/aws/amazon-ecs-agent" --privileged "amazon/amazon-ecs-agent-test:make"
 
-run-functional-tests: test-registry
+run-functional-tests: docker test-registry
```

Comment:
Yeah, @samuelkarp  pointed that out as well. I was unaware of the env var for that. I'll revert this or create another target which can be used during development to do this.